### ISSUE TITLE
Article navigation bar is unresponsive

### ISSUE THREAD

**Issue by jcp-sd** [FredJul/Flym#675]
There seems to be a regression of sorts on my device with the recent update (Flym 2.5.1)

When reading a feed entry, the toolbar on top is now very unresponsive. With the exception of the menu button (the 3 vertical dots), _all of the other buttons_ are very difficult to activate (back arrow at left, star to mark as favorites, get full content, and read in browser buttons).

If one is persistent, I can get them to eventually activate, but sometimes it takes as many as 15 to 20 taps! This was never an issue in older releases on my device?

An icon mapping issue of sorts? (tapping slightly below the button seems to help)

Android 7.1.1

**Comment by nvgtx** [comment]
I second that.

Android 7.1.2
Flym 2.5.2

**Comment by infinitytec** [comment]
Can confirm that as well.
OneUI and Android 9

Flym 2.5.2

**Comment by erazemkokot** [comment]
In my everything in the top bar works except the back button.

Android 10
Flym 2.5.2

**Comment by infinitytec** [comment]
Yeah I think it's just the back button for me. Sometimes it works fine but other times I need to tap it several times.

**Comment by int-ua** [comment]
Workaround is to mark as unread and swipe to mark as read again. Any clues about the reason?

**Comment by erazemk** [comment]
> Workaround is to mark as unread and swipe to mark as read again. Any clues about the reason?

The reason this works is that the problem only occurs with the buttons in the reading view, what you recommended is to go out of the reading view, where the problem doesn't occur.

If what you intended was to get back to the article list after reading an article you can press the back button to do that.

**Comment by int-ua** [comment]
> If what you intended was to get back to the article list after reading an article you can press the back button to do that.

I don't follow. You said yourself that the back button is not working:

> In my everything in the top bar works except the back button.

**Comment by erazemk** [comment]
> I don't follow. You said yourself that the back button is not working

Sorry, forgot to specify. The software back button in the top bar doesn't work, but if you wanna go back after reading the article you can press your phone's hardware back button, that still works.

It doesn't fix the problem, but if you wanted a shorter way to exit the reading mode without having to do the steps you recommended, that is what you can do.

**Comment by int-ua** [comment]
Oh, yeah, I became so accustomed to the software back button that I didn't even think about the hardware one, thank you. Also, at first it was intentional to reduce hardware wear.
